cassandra-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/BatchCassandraSource.java

```diff
@@ -111,7 +114,7 @@ public void prepareRun(BatchSourceContext context) throws Exception {
     }
     CqlConfigHelper.setInputCql(conf, config.query);
 
-    job.setInputFormatClass(CqlInputFormat.class);
+    context.setInput(Input.of("Cassandra", new SourceInputFormatProvider(CqlInputFormat.class, conf)));
```

Comment:
Hmm, I made and merged this change in my PR https://github.com/caskdata/hydrator-plugins/pull/235. Not sure why it is not showing up in the diff?